sdks/uniswapx-sdk/integration/test/PriorityOrder.spec.ts

```diff
@@ -0,0 +1,723 @@
+import hre, { ethers } from "hardhat";
+import { mine } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { BigNumber, Signer, Wallet } from "ethers";
+
+import { BlockchainTime } from "./utils/time";
+
+import PriorityOrderReactorAbi from "../../abis/PriorityOrderReactor.json";
+import Permit2Abi from "../../abis/Permit2.json";
+import MockERC20Abi from "../../abis/MockERC20.json";
+
+import { Permit2, PriorityOrderReactor, MockERC20 } from "../../src/contracts";
+import { PriorityOrderBuilder, PriorityCosignerData } from "../../dist/src";
+
+describe("PriorityOrder", () => {
```

Comment:
Should we add tests around the priority fee buffer (min priority fee)?

Reply:
skipping; see other comment